docs/tutorials/deploy-multi-cloud-application.md

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_position: 1
+---
+
+# Deploying Multi-Cloud Applications
```

Comment:
We use `Deploy`/`Create` in other tutorial titles, Can we keep consistency? Not sure which is better